zest/releaser/pypi.py

```diff
@@ -310,7 +365,10 @@ def extra_message(self):
             [zest.releaser]
             extra-message = [ci skip]
         """
-        return self.__get_message_config__("extra-message")
+        try:
+            return self.config["extra-message"]
+        except KeyError:
+            return None
```

Comment:
I dislike using exceptions when not needed :-) The following is simpler, I think:

```suggestion
        return self.config.get("extra-message")
```

(Same for the other instances).

Reply:
Thanks, I've done this now in https://github.com/zestsoftware/zest.releaser/pull/415/commits/41788f5719d2b4b2a8a4ec65a718cae732831601.